securesystemslib/gpg/functions.py

```diff
@@ -299,3 +299,29 @@ def export_pubkey(keyid, homedir=None):
   key_bundle = securesystemslib.gpg.common.get_pubkey_bundle(key_packet, keyid)
 
   return key_bundle
+
+
+def export_pubkeys(keyids, homedir=None):
+  """Export multiple public keys from a GnuPG keyring.
+
+  Arguments:
+    keyid: A list of OpenPGP keyids in KEYID_SCHEMA format.
+    homedir (optional): A path to the GnuPG home directory. If not set the
+        default GnuPG home directory is used.
+
+  Raises:
+    TypeError: Keyids is not iterable.
+    See 'export_pubkey' for other exceptions.
+
+  Returns:
+    A dict with OpenPGP keyids for dict keys and keys in GPG_PUBKEY_SCHEMA
+    format for values.
```

Comment:
I suggest clarifying that the keys of the returned dict are the keyids that were passed as an argument.

```suggestion
    A dict with the OpenPGP keyids passed as the keyids argument for dict keys
    and keys in GPG_PUBKEY_SCHEMA format for values.
```